Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "structured"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
Since any version change is supposed to be decided by the master, revert this please.